src/integ-test/bdd-test/resources/config.json

```diff
@@ -2,21 +2,13 @@
   "_comment": "this file should be generated by test caller in order to configure systems and variables environment",
   "services": {
     "nodo-dei-pagamenti": {
-      "url": "http://localhost:8081",
-      "healthcheck": "/monitor/health",
+      "url": "https://soap-input-uat-npa-nodopagamenti.tst-npc.sia.eu/uat",
```

Comment:
@aomegax i think we can remove `config.json` and leave only `config.json.example` and mention it, in to `README` file